Content.Client/Examine/ExamineButton.cs

```diff
@@ -27,14 +23,16 @@ public sealed class ExamineButton : ContainerButton
     public TextureRect Icon;
 
     public ExamineVerb Verb;
+    private SpriteSystem _spriteSystem;
```

Comment:
```suggestion
    private SpriteSystem _sprite;
```
naming convention
same below